styleguide/writing-style.md

```diff
@@ -0,0 +1,183 @@
+# Writing style guide
+
+Welcome to the Buildkite writing style guide.
+
+These guidelines provide details about the language and words used to write the Buildkite docs, as well as details about its writing style and format.
+If something isn't included in this guide, see the [Google developer documentation style guide](https://developers.google.com/style), followed by the [Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/welcome/).
+
+For details about the Markdown syntax used to render your writing as well as the Buildkite docs' file structure, as well as details on working with the site, and using screenshots, refer to the [Markdown syntax style guide](markdown-syntax-style.md).
+
+Table of contents (main headings):
+* [Language](#language)
+* [Style and formatting](#style-and-formatting)
+
+## Language
+
+This section covers everything related to the language, words, and formatting used in Buildkite documentation (including the website, to some extent).
+
+### English flavor
+
+Use US English. As a highly multi-national team, here is the list of the most notable differences between [Australian, American (US), and British English](https://blog.e2language.com/australian-english-vs-american-english-vs-british-english/) English to watch out for.
+
+### Dictionary
+
+Buildkite's first-call dictionary is [Merriam Webster](https://www.merriam-webster.com/) for US English. This is not a single source of truth, just a good starting/reference point.
+
+> [!NOTE]
+> The Buildkite docs build process uses [Vale](https://github.com/apps/vale-linter), a [linter](/README.md#linting) with a file that contains a [custom list of words](/vale/styles/vocab.txt) that may confuse the spellchecker, [filename linter](https://ls-lint.org/1.x/getting-started/introduction.html), and a [Markdown linter](https://github.com/DavidAnson/markdownlint).
+
+### Commas
+
+Keep it clean and uncluttered. If the sentence can live without that comma - leave it out.
```

Comment:
Is this right? The second sentence seems like a comma would be more appropriate?

This also feels a bit strange as guidance beside a point about using serial commas. I wonder if it should be something that emphasizes clarity not clutter, like:

> Keep it clean and uncluttered. Use commas for clarity, not decoration.

Reply:
I like that, less contradictory with the next paragraph.

I've decided to remove the distinction between the 'Commas' and 'Serial commas' sections and place all this content under a single heading 'Commas'.